### DISCUSSION TITLE
建议没用jpa的项目就不要参考这个例子,尤其是mybatis

### DISCUSSION THREAD

**Discussion by ddddage** [citerus/dddsample-core#86] [General]
用mybatis cpoy一份这个项目就像用橙子做橙子的分子料理一样,拿个橙子榨橙汁,再向橙汁里加点淀粉白面复原成橙子,说没有吧还有点用,说有用吧是真没用

**Comment by littelrain29** [comment]
即使用mybatis的项目也可以参考这个项目，毕竟从数据库层面来讲，它们使用同等的结构。 （由于语言问题， 此评论为代写！）

**Reply by ddddage** [reply]
你说的对,但是在维护多对多结构的时候用jpa可以专注实体与实体的关联,但是如果用mybatis还是要实现具体的表与表的关联关系,我觉得这会让开发和维护过程变得不愉快,过去我一直用mybatis,对jpa感觉一般,但是看到这个例子我觉得jpa和mybatis的初始思路完全不一样,从我直观感受来说jpa和这个项目结合的很自然,而mybatis和jpa比较就像是人的原生四肢和人造假肢, 假肢能工作而且在某些地方比人的原生四肢功能更强大,但是总体来说原生的四肢比假肢感觉更好

**Comment by chenqingze** [comment]
Domain entity != JPA entity

Domain entity != data mapping object 

The sample is about DDD,not about ORM!

**Comment by mackapappa** [comment]
Closing this discussion since it's off topic.
